roles/vault/tasks/main.yml

```diff
@@ -2,6 +2,9 @@
 - name: "Vault Prechecks"
   import_tasks: prechecks.yml
 
+- name: "Vault Prerequisites"
+  import_tasks: prereqs.yml
```

Comment:
Do we really need that in a separate file? It's not really a prerequisite.

Reply:
It is, isn't it?

Reply:
Well, maybe it is, but does it deserve a separate file? If we would have more - sure.

Reply:
I asked myself the same question, but it's not a prechecks and we need it for both consul and vault. I could have added it at the beginning of the consul play, but separating it seemed more logical.